stackslib/src/chainstate/stacks/boot/pox_4_tests.rs

```diff
@@ -3282,6 +3283,323 @@ fn stack_agg_commit_verify_sig() {
+        let principal = PrincipalData::from(address.clone());
+        let nonce = 0;
+        Self {
+            private_key,
+            public_key,
+            address,
+            principal,
+            pox_address,
+            nonce,
+        }
+    }
+}
+
+/// Helper function to advance to a specific block height with the passed txs as the first in the block
+/// Returns a tuple of the tip and the observed block that should contain the provided txs
+fn advance_to_block_height(
+    peer: &mut TestPeer,
+    observer: &TestEventObserver,
+    txs: &[StacksTransaction],
+    peer_nonce: &mut usize,
+    target_height: u64,
+) -> (StacksBlockId, TestEventObserverBlock) {
+    let mut tx_block = None;
+    let mut latest_block = None;
+    let mut passed_txs = txs;
+    while peer.get_burn_block_height() < target_height {
+        latest_block = Some(peer.tenure_with_txs(&passed_txs, peer_nonce));
+        passed_txs = &[];
+        if tx_block.is_none() {
+            tx_block = Some(observer.get_blocks().last().unwrap().clone());
+        }
+    }
+    let latest_block = latest_block.expect("Failed to get tip");
+    let tx_block = tx_block.expect("Failed to get tx block");
+    (latest_block, tx_block)
+}
+
+#[test]
+/// Test for verifying that the stacker aggregation works as expected
+///   with new signature parameters.
+fn stack_agg_increase() {
+    // Alice service signer setup
+    let mut alice = StackerSignerInfo::new();
+    // Bob pool operator
+    let mut bob = StackerSignerInfo::new();
+    // Carl pool delegate
+    let mut carl = StackerSignerInfo::new();
+    // Dave pool delegate
+    let mut dave = StackerSignerInfo::new();
+    // Eve late pool delegate
+    let mut eve = StackerSignerInfo::new();
+
+    let default_initial_balances = 1_000_000_000_000_000_000;
+    let observer = TestEventObserver::new();
+    let test_signers = TestSigners::default();
+    let mut initial_balances = vec![
+        (alice.principal.clone(), default_initial_balances),
+        (bob.principal.clone(), default_initial_balances),
+        (carl.principal.clone(), default_initial_balances),
+        (dave.principal.clone(), default_initial_balances),
+        (eve.principal.clone(), default_initial_balances),
+    ];
+    let aggregate_public_key = test_signers.aggregate_public_key.clone();
+    let mut peer_config = TestPeerConfig::new(function_name!(), 0, 0);
+    let private_key = peer_config.private_key.clone();
+    let addr = StacksAddress::from_public_keys(
+        C32_ADDRESS_VERSION_TESTNET_SINGLESIG,
+        &AddressHashMode::SerializeP2PKH,
+        1,
+        &vec![StacksPublicKey::from_private(&private_key)],
+    )
+    .unwrap();
+
+    // reward cycles are 5 blocks long
+    // first 25 blocks are boot-up
+    // reward cycle 6 instantiates pox-3
+    // we stack in reward cycle 7 so pox-3 is evaluated to find reward set participation
```

Comment:
Are these copied from somewhere? I don't think they are correct here.

Reply:
Yeap, done in #4586 with the original setup coming from 'test_scenario_five.'

Reply:
I would just delete those four lines of comments (they were reasoning about a calculation that's no longer done here).

Reply:
Agreed.